connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsIntegrationTransactionsTest.java

```diff
@@ -45,7 +45,13 @@ public void startClusters() throws Exception {
         backupBrokerProps.put("transaction.state.log.min.isr", "1");
         producerProps.put(ProducerConfig.ENABLE_IDEMPOTENCE_CONFIG, "true");
         producerProps.put(ProducerConfig.TRANSACTIONAL_ID_CONFIG, "embedded-kafka-0");
-        super.startClusters();
+        startClusters(new HashMap<String, String>() {{
+                put("topics", "test-topic-.*, primary.test-topic-.*, backup.test-topic-.*");
+                put(PRIMARY_CLUSTER_ALIAS + "->" + BACKUP_CLUSTER_ALIAS + ".enabled", "true");
+                put(BACKUP_CLUSTER_ALIAS + "->" + PRIMARY_CLUSTER_ALIAS + ".enabled", "true");
+                // This is not necessary for this test, but is not tested elsewhere.
+                put("offset.lag.max", "0");
```

Comment:
Don't we assume that the value of this property is non-zero in `MirrorConnectorsIntegrationBaseTest::waitForConsumerGroupFullSync`?

Reply:
The way the assertions were written, they would still be correct for offset.lag.max=0, just weaker than necessary.
In the new parameterization, if we run with offset.lag.max=0, the assertions will also get stronger to match.